### ISSUE TITLE
Puked on ES 6 object deconstruction:

### ISSUE THREAD

**Issue by Samsinite** [abuiles/ember-watson#76]
```
Oh Dear! I wasn't able to save the following files:
app/controllers/sign-up.js - Line 3: Unexpected identifier

A possible cause is having all the source code commented.
If that's not the problem please fill a report with the hospital directors
at https://github.com/abuiles/ember-watson/issues.
```

Offending line is:
```
const { get, set } = Ember;
```

**Comment by marcioj** [comment]
This is a dup of #70. And it's already fixed on master. I'm creating a release right now.

**Comment by marcioj** [comment]
Released `0.6.3`. Thanks for open the issue!

**Comment by Samsinite** [comment]
Nice! Thanks for the follow up.
